PresentationLayer/Sources/PresentationLayer/Features/NextToGo/NextToGoViewModel.swift

```diff
@@ -148,3 +190,50 @@ private extension Race {
         return list
     }
 }
+
+enum Country: String, CaseIterable, Identifiable {
+
+    var id: String { rawValue }
+
+    /// Used as helper to combine all countries internally
+    /// and treat that as an item in the picker to select
+    case international = "INTL"
+
+    case aus =  "AUS"
+    case nz =   "NZL"
+    case jpn =  "JPN"
+    case ind =  "IND"
+    case uk =   "GBR"
+    case fra =  "FRA"
+    case brz =  "BRA"
+    case usa =  "USA"
+
+    var name: String {
+        switch self {
+        case .international:    return "INTL"
+        case .aus:              return "Australia"
+        case .nz:               return "New Zealand"
+        case .jpn:              return "Japan"
+        case .ind:              return "India"
+        case .uk:               return "United Kingdom"
+        case .fra:              return "France"
+        case .brz:              return "Brazil"
+        case .usa:              return "USA"
+        }
+    }
+
+    var fullDisplayName: String {
+        if self == .international {
+            return "🌏" + " " + name
```

Comment:
Maybe use a mini function such as:
```
func internatonalDisplayName(_ name: String) -> String {
   "🌏" + " " + name
}
``` 

And re-use it twice in this area. :)

Reply:
Here you go Arinjoy, pushed with some tweaks and a11y VO updates.
80ef992d8cefe11275e0116aaea82a5cffbcdae1